lib/lita/adapters/slack/api.rb

```diff
@@ -70,15 +74,22 @@ def set_topic(channel, topic)
         end
 
         def rtm_start
-          response_data = call_api("rtm.start")
+          rtm_response_data = call_api("rtm.connect")
```

Comment:
Why prefix it with rtm?
```suggestion
          response_data = call_api("rtm.connect")
```